core/src/main/java/com/sopt/core/designsystem/component/timetable/NoostakEditableTimeTable.kt

```diff
@@ -0,0 +1,156 @@
+package com.sopt.core.designsystem.component.timetable
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.defaultMinSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.grid.GridCells
+import androidx.compose.foundation.lazy.grid.LazyVerticalGrid
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.theme.NoostakTheme
+import com.sopt.core.extension.noRippleClickable
+import com.sopt.domain.entity.AvailableTimeEntity
+import com.sopt.domain.entity.TimeEntity
+import com.sopt.domain.entity.TimeTableEntity
+
+@Composable
+fun NoostakEditableTimeTable(
+    data: TimeTableEntity,
+    modifier: Modifier = Modifier,
+    onSelectionChange: (List<TimeEntity>) -> Unit
+) {
+    val days = data.timeEntity.size
+    val timeSlots = calculateTimeSlots(data.startTime, data.endTime)
+    val selectedCells = remember { mutableStateListOf<Pair<Int, Int>>() } // Row, Column 저장
+
+    LazyVerticalGrid(
+        modifier = modifier
+            .border(
+                width = 1.dp,
+                color = NoostakTheme.colors.gray200,
+                shape = RoundedCornerShape(8.dp)
+            ),
+        columns = GridCells.Fixed(days + 1)
+    ) {
+        items((days + 1) * (timeSlots + 1)) { index ->
+            val rowIndex = index / (days + 1)
+            val columnIndex = index % (days + 1)
```

Comment:
P2: 요렇게 Pair 객체를 생성하면 더 간단하게 사용 가능합니다!
+) 보니까 selectedCells도 Pair 객체 사용하셨네요~! 👍👍

```suggestion
val (rowIndex, columnIndex) = index / (days + 1) to index % (days + 1)
```

> - **Pair** : 두 개의 값을 하나의 객체로 묶는 클래스
> - first와 second라는 두 개의 값을 저장함 (서로 다른 타입 가능)

Reply:
옴뫄 요로코롬 만들수있구먼